tests/suites/test_suite_ssl.function

```diff
@@ -69,9 +69,100 @@ exit:
 
 typedef enum {
     RECOMBINE_NOMINAL,          /* param: ignored */
+    RECOMBINE_SPLIT_FIRST,      /* param: offset of split (<=0 means from end) */
+    RECOMBINE_INSERT_EMPTY,     /* param: offset (<0 means from end) */
     RECOMBINE_COALESCE,         /* param: min number of records */
+    RECOMBINE_COALESCE_SPLIT_ONCE, /* param: offset of split (<=0 means from end) */
+    RECOMBINE_COALESCE_SPLIT_ENDS, /* the hairiest one? param: offset, must be >0 */
```

Comment:
I trust you that's the hairiest one but I can't understand what it does just from reading the name and comment. Edit after reading the rest of the commit: `SPLIT_BOTH_ENDS` would be more meaningful to me.

Reply:
Ah, completely missed the pun, sorry.